<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seismic Viewer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Show columns</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="cbType">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbFrequency">
           <property name="text">
            <string>Frequency</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbTimeStart">
           <property name="text">
            <string>Time start</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbTimeStop">
           <property name="text">
            <string>Time stop</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbDuration">
           <property name="text">
            <string>Duration</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbLongitude">
           <property name="text">
            <string>Longitude</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbLatitude">
           <property name="text">
            <string>Latitude</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bApplyColumnsVisible">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QTableWidget" name="gFileInfo">
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>File name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Frequency</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time stop</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Longitude</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Latitude</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="bAddFiles">
          <property name="toolTip">
           <string>Add new files</string>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>➕</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bDelRow">
          <property name="toolTip">
           <string>Remove current file</string>
          </property>
          <property name="text">
           <string>➖</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bRemoveFiles">
          <property name="toolTip">
           <string>Clear list</string>
          </property>
          <property name="text">
           <string>❌</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bSaveToFile">
          <property name="text">
           <string>Export table to file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="aSpectrograms"/>
    <addaction name="aRevise"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>&amp;Utilities</string>
    </property>
    <addaction name="aFilesJoining"/>
    <addaction name="aFileHeaderRedactor"/>
   </widget>
   <addaction name="menuProcessing"/>
   <addaction name="menuUtilities"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aSpectrograms">
   <property name="text">
    <string>&amp;Spectrograms</string>
   </property>
  </action>
  <action name="aRevise">
   <property name="text">
    <string>&amp;Revise</string>
   </property>
  </action>
  <action name="aLoadFiles">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="aCheckArchive">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="aUnpackArchive">
   <property name="text">
    <string>Unpack</string>
   </property>
  </action>
  <action name="aCreateArchive">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="aCheckingArchive">
   <property name="text">
    <string>Checking</string>
   </property>
  </action>
  <action name="aFilesJoining">
   <property name="text">
    <string>&amp;Files joining</string>
   </property>
  </action>
  <action name="aSpectrumViewer">
   <property name="text">
    <string>&amp;Spectrum Viewer</string>
   </property>
  </action>
  <action name="aFileHeaderRedactor">
   <property name="text">
    <string>File header redactor</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>gFileInfo</tabstop>
  <tabstop>cbFrequency</tabstop>
  <tabstop>cbTimeStart</tabstop>
  <tabstop>cbTimeStop</tabstop>
  <tabstop>cbDuration</tabstop>
  <tabstop>cbLongitude</tabstop>
  <tabstop>cbLatitude</tabstop>
  <tabstop>bApplyColumnsVisible</tabstop>
  <tabstop>bAddFiles</tabstop>
  <tabstop>bDelRow</tabstop>
  <tabstop>bRemoveFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
